Proteomics/ProteolyticDigestion/Protease.cs

```diff
@@ -172,21 +167,19 @@ public CleavageSpecificity GetCleavageSpecificity(string proteinSequence, int st
         internal List<int> GetDigestionSiteIndices(string proteinSequence)
         {
             var indices = new List<int>();
-            for (int i = 0; i < proteinSequence.Length - 1; i++)
+            for (int i = 0; i < proteinSequence.Length; ++i)
             {
-                foreach (var c in SequencesInducingCleavage)
+                foreach(var c in MotifList)
```

Comment:
much cleaner. thx. this looks good. I have some minor code style comments we should talk about but this is good

Reply:
also need to redo the benchmarks for time and number of peptides

Reply:
I did the benchmark test yesterday for 4 different protease and it all looks good

Reply:
awesome 👍